packages/python-google-compute-engine/google_compute_engine/networking/network_setup/tests/network_setup_test.py

```diff
@@ -67,6 +67,77 @@ def testEnableIpv6(self, mock_call):
     ]
     self.assertEqual(mocks.mock_calls, expected_calls)
 
+  @mock.patch('google_compute_engine.networking.network_setup.network_setup.subprocess.check_call')
+  def testDisableIpv6(self, mock_call):
+    mocks = mock.Mock()
+    mocks.attach_mock(mock_call, 'call')
+    mocks.attach_mock(self.mock_logger, 'logger')
+    mocks.attach_mock(self.mock_distro_utils.EnableIpv6, 'enable')
+    mocks.attach_mock(self.mock_distro_utils.DisableIpv6, 'disable')
+    expected_calls = []
+
+    # Clean run, run disable once e.g. at boot.
+    network_setup.NetworkSetup.DisableIpv6(
+        self.setup, ['A'])
+    self.assertEqual(self.setup.ipv6_interfaces, set([]))
+    # No more disables allowed, have to follow the contract of Enable and then
+    # Disable.
+    network_setup.NetworkSetup.DisableIpv6(
+        self.setup, ['A'])
+    expected_calls.extend(
+        [mock.call.logger.info(mock.ANY, ['A']),
+         mock.call.disable(['A'], mock.ANY)])
+    # Enable interfaces.
+    network_setup.NetworkSetup.EnableIpv6(self.setup, ['A', 'B', 'C'])
+    expected_calls.extend(
+        [mock.call.logger.info(mock.ANY, ['A', 'B', 'C']),
+         mock.call.enable(['A', 'B', 'C'], mock.ANY,
```

Comment:
Please line break after the first open paren (everywhere).

Reply:
Is tox -e lint not a valid linter ? Possibly the lint rules need to be updated ? Since before I send it for review I run tox -e lint and make sure there are no errors.

Reply:
We disable some rules since we use google style instead of pep 8 (in particular, 80 character limit instead of 79).